Makefile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+GIT_COMMIT ?= $(shell git rev-parse HEAD)
+
+LDFLAGS?='-extldflags "-static"'
+GO_FILES=$(shell go list ./...)
+
+REPO=gmontero
```

Comment:
This seems like it should be changed

Reply:
Should probably be "openshift" ?

Reply:
it will once we start producing images in CI / ART ... in the interim, we are using my repo

Reply:
I'll put a comment stating as such